build/charts/antrea-windows/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Do we need this file? I suppose most of files and directories are from your local? 
Please double check and remove unnecessary lines.

Reply:
It is the default ignore file created by helm when creating a new chart. The main antrea chart also used the same file so I did not think much about it.